prometheus/src/cluster_metrics.rs

```diff
@@ -52,5 +77,50 @@ pub fn write_cluster_metrics<W: io::Write>(
         },
     )?;
 
+    // Vote accounts information
```

Comment:
Hmm yeah, so now we run into this issue that we have the structure of our metrics “inside out” ... It is a bit ugly indeed ... I need to think about it a bit more, I have a feeling that an elegant solution exists.